llvm/lib/Support/LockFileManager.cpp

```diff
@@ -189,6 +189,10 @@ Expected<bool> LockFileManager::tryLock() {
     return createStringError(EC, "failed to create unique file " +
                                      UniqueLockFileName);
 
+  // Clean up the unique file on signal or scope exit. This also releases the
+  // lock if it's held since the .lock symlink will point to a nonexistent file.
```

Comment:
I know you're just moving this comment, but I think it's wrong. On Windows `create_link` makes a hard link, so removing the unique name does nothing. I also think it doesn't really matter since `readLockFile` will remove the old one if the process is dead anyway.

Reply:
Good observation. I'll remove the second sentence before landing.